fvtest/tril/tril/ilgen.cpp

```diff
@@ -248,6 +248,48 @@ TR::Node* Tril::TRLangBuilder::toTRNode(const ASTNode* const tree) {
          node = TR::Node::create(opcode.getOpCodeValue(), childCount);
          node->setBranchDestination(targetEntry);
      }
+     else if (opcode.isCall()) { 
```

Comment:
Please add a trace message explaining that a `call` opcode has been found (as is done for other cases).

Reply:
✅